src/lib.rs

```diff
@@ -43,7 +43,7 @@ cfg_if! {
     }
 }
 
-pub(crate) type HashMap<K, V, S> = std::collections::HashMap<K, SharedValue<V>, S>;
```

Comment:
This type alias is exposed by public api's and therefor is a semver breakage, this was experienced in using one of the rust-analyzer crates as a dependency https://github.com/rust-lang/rust-analyzer/issues/12344#issuecomment-1151259946

Reply:
@Veykril Hey! This is indeed a semver breakage, however this is only visible if you use the `raw_api` feature which exposes the raw shards. I should probably do a better job at noting this but this feature is unstable, this means you need to pin the versions if you rely on the types from that API as it may change due to internal needs like this PR and an internal lock change that was also done recently.

Reply:
I see, good to know!